sdk/daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -111,4 +111,5 @@ message FatContractInstance {
 message KeyWithMaintainers {
   bytes key = 1;
   repeated string maintainers = 2; // the maintainers induced by the key
+  string package_name = 3;
```

Comment:
You should be able to get the package name from the node, like we get the template ID